components/buttons_react.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export const InputField = ( ({onValueChange}) => (
+    <input type="text"
+    placeholder="What to do?"
+    className="textBox"
+    onChange={onValueChange}/>
+));
+
+export const InputButton = ( ({postData}) => (
```

Comment:
Move `InputButton` and `InputField` to `TodoForm` component.
Usually it is overhead to create React component for single HTML tag if it has no additional logic